cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -399,6 +400,7 @@ protected void configure() {
       handlerBinder.addBinding().to(AppLifecycleHttpHandler.class);
       handlerBinder.addBinding().to(DashboardHttpHandler.class);
       handlerBinder.addBinding().to(ProgramLifecycleHttpHandler.class);
+      handlerBinder.addBinding().to(OperationsDashboardHttpHandler.class);
```

Comment:
Please open a JIRA that this should be moved out of app-fabric. Once we have the proper API for system app, this should be part of the reporting system app.